cpg-language-jvm/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,14 @@
+<Configuration status="WARN">
+    <Appenders>
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss,SSS} %-5p %C{1} %m%n"/>
+            <ThresholdFilter level="TRACE"/>
```

Comment:
We use the configuration inconsistently across frontends. java and c++ use DEBUG for the appender. go, python and jvm use trace. We should discuss this.

Reply:
I think trace makes sense for the appender, this is just the MAXIMUMST log level and a few lines below the actual log level is set to DEBUG.